### PR TITLE
Feature/gh48 span interceptor

### PR DESCRIPTION
**Pull request by jlupi**

### CODE REVIEW

File: trasier-client-api/src/main/java/com/trasier/client/api/TrasierConstants.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
     String HEADER_CONVERSATION_ID = "X-Conversation-Id";
     String HEADER_TRACE_ID = "X-Trace-Id";
     String HEADER_SPAN_ID = "X-Span-Id";
+    String HEADER_SAMPLE_FLAG = "X-Sample";
```
Comment (codepfleger) [inline trasier-client-api/src/main/java/com/trasier/client/api/TrasierConstants.java]: Better X-Conversation-Sample

File: trasier-client-opentracing-spring-interceptor/src/main/java/com/trasier/opentracing/spring/interceptor/InterceptorWSConfiguration.java [hunk 2]
```diff
@@ -28,6 +29,9 @@
     @Autowired(required = false)
     private Set<WebServiceGatewaySupport> webServiceGatewaySupports;
 
+    @Autowired(required = false)
+    private TrasierOperationFilter operationFilter;
```
Comment (codepfleger) [inline trasier-client-opentracing-spring-interceptor/src/main/java/com/trasier/opentracing/spring/interceptor/InterceptorWSConfiguration.java]: Why is it only operation-specific and not something general?

File: trasier-client-opentracing/src/main/java/com/trasier/client/opentracing/TrasierSpan.java [hunk 3]
```diff
@@ -22,7 +22,7 @@ public TrasierSpan(Client client, TrasierClientConfiguration configuration, com.
 
     @Override
     public SpanContext context() {
-        return new TrasierSpanContext(wrapped.getConversationId(), wrapped.getTraceId(), wrapped.getId(), baggageItems);
+        return new TrasierSpanContext(wrapped.getConversationId(), wrapped.getTraceId(), wrapped.getId(), !wrapped.isCancel(), baggageItems);
```
Comment (codepfleger) [inline trasier-client-opentracing/src/main/java/com/trasier/client/opentracing/TrasierSpan.java]: cancel=sampled?

File: trasier-client-api/src/main/java/com/trasier/client/api/TrasierConstants.java (hunk 1 above)
Comment (jlupi) [reply trasier-client-api/src/main/java/com/trasier/client/api/TrasierConstants.java]: ok

File: trasier-client-opentracing/src/main/java/com/trasier/client/opentracing/TrasierSpan.java (hunk 3 above)
Comment (jlupi) [reply trasier-client-opentracing/src/main/java/com/trasier/client/opentracing/TrasierSpan.java]: cancel = !sampled

**Pull request merged**